yb-voyager/cmd/importSchema.go

```diff
@@ -47,8 +47,12 @@ var importSchemaCmd = &cobra.Command{
 		if tconf.TargetDBType == "" {
 			tconf.TargetDBType = YUGABYTEDB
 		}
+		err := retrieveMigrationUUID()
```

Comment:
Might be slightly unrelated question, but at this point what if export-dir is empty and user runs import-schema? What happens then?
Same question for import-data,  analyze-schema as well..

Reply:
In case of empty dirs and these commands we already have other checks just before this that checks if export-dir is initiated. And we can't do much in those cases we will still get the 00000... uuid in callhome

Reply:
got it, but i guess now we will get the error message, so that should be helpful.